### PR TITLE
Add local DA guide - a.k.a. Hello World Tutorial

### PR DESCRIPTION
**Pull request by jcstein**
Resolves https://github.com/rollkit/docs/issues/92

### CODE REVIEW

File: docs/tutorials/local-da-devnet.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+this process can be done on any machine of your choosing. We tested it out on a machine with the following specs:
+
+- Memory: 1 GB RAM
+- CPU: Single Core AMD
+- Disk: 25 GB SSD Storage
+- OS: Ubuntu 22.10 x64
+
+## 🌕 Running Local Devnet with a Rollkit Rollup
+
+First, run the [`local-celestia-devnet`](https://github.com/celestiaorg/local-celestia-devnet) by running the following command:
```
Comment (YazzyYaz) [inline docs/tutorials/local-da-devnet.md:33]: Missing a prerequisites section to install Docker

**Review by YazzyYaz — COMMENTED** [review]
Left a comment.

Only other feedback is that maybe this just be labeled a "Hello World" tutorial

File: docs/tutorials/local-da-devnet.md (hunk 1 above)
Comment (jcstein) [reply docs/tutorials/local-da-devnet.md:33]: awesome, thanks yaz

Comment (jcstein) [reply docs/tutorials/local-da-devnet.md]: 31f5f0c324cb775b85522a51647f5ba65fc67350

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**